ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot

```diff
@@ -220,9 +225,40 @@ Test Tensorflow Model Grpc Inference Via UI (Triton on Kserve)    # robocop: off
     ...  AND
     ...  Delete Serving Runtime Template From CLI    displayed_name=triton-tensorflow-grpc
 
+Test KERAS Model Rest Inference Via UI(Triton on Kserve)
+    [Documentation]    Test the deployment of an keras model in Kserve using Triton
+    [Tags]    Tier2      RHOAIENG-10328
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    ...    existing_project=${FALSE}
+    Open Dashboard Settings    settings_page=Serving runtimes
+    Upload Serving Runtime Template    runtime_filepath=${KERAS_RUNTIME_FILEPATH}
```

Comment:
as there are other tests which verify the flow from the ui, wouldn't it be simpler run on the backend only?

Reply:
@rnetser These are the pending PRs. Since we have a few more models left, we will proceed with API-based testing instead of UI-based testing. We have already discussed this with Tarun.